### ISSUE TITLE
iOS 10.2 - Returning nil in defaultRegionCode

### ISSUE THREAD

**Issue by Sol88** [marmelroy/PhoneNumberKit#96]
This method
```
public class func defaultRegionCode() -> String
```

returns nil in iOS 10.2

**Comment by Sol88** [comment]
I think the problem is here 
```
public class func defaultRegionCode() -> String {
...
        let currentLocale = Locale.current
        if #available(iOS 10.0, *) {
            let countryCode = currentLocale.regionCode
            return countryCode?.uppercased() ?? ""
...
}
```

maybe it should be better 

```
public class func defaultRegionCode() -> String {
...
        let currentLocale = Locale.current
        if #available(iOS 10.0, *) {
            let countryCode = currentLocale.regionCode
            if let countryCode = countryCode?.uppercased() {
                return countryCode.uppercased()
            }
...
}
```

so if this condition fails we will return 

`return PhoneNumberConstants.defaultCountry`

**Comment by marmelroy** [comment]
Thanks a lot for catching this @Sol88. Indeed, it shouldn't fall back to an empty string.
